packages/expo-image-picker/CHANGELOG.md

```diff
@@ -14,7 +14,9 @@
 
 ### 🎉 New features
 
-- On iOS 14+ added support for selecting multiple images/videos. ([#18102](https://github.com/expo/expo/pull/18102) by [@barthap](https://github.com/barthap))
+- On iOS 14+ added support for selecting multiple images/videos. ([#18102](https://github.com/expo/expo/pull/18102), [#18138](https://github.com/expo/expo/pull/18138), [#18145](https://github.com/expo/expo/pull/18145) by [@barthap](https://github.com/barthap))
```

Comment:
Was it intentional to put entries under already released v`13.2.0`? 😛

Reply:
Arrgh, glad you noticed that. Publishing in the meantime broke all my changelogs 😠 
I'll move everything to unpublished even if that's not really true. I don't care.

Reply:
Maybe that's something that the code reviewing bot could detect easily 🤔

Reply:
I think you're right, it is. I can take care of that.